applications/serial_lte_modem/src/slm_at_host.c

```diff
@@ -427,19 +424,32 @@ static void format_final_result(char *buf, size_t buf_len, size_t buf_max_len)
 	}
 }
 
-static void cmd_send(uint8_t *buf, uint16_t cmd_length, uint16_t buf_size)
+static size_t cmd_name_toupper(char *cmd, size_t cmd_len)
+{
+	size_t i = 0;
+
+	/* Set/test command names are delimited by '=', read by '?'. */
+	while (i != cmd_len && cmd[i] != '=' && cmd[i] != '?') {
```

Comment:
Would it be better to utilize for?

Reply:
Looks bit weird with while, so a slight preference on for. But you can also resolve this if you want.